README.md

```diff
@@ -56,6 +57,13 @@ set :deploy_via, :jenkins_artifact
 * xz
 * raw
 
+### `release_name_from` option
+
+You can set either `'build_at'` (default) or `'deploy_at'` to this option.
```

Comment:
In my image, receiving symbols can accept limited values(for example, `:sftp` or `:ftp`, [like this](https://github.com/capistrano/capistrano-2.x-docs/blob/master/2.x-DSL-Action-File-Transfer-Download.md) ) and receiving strings can accept any string values.